### PR TITLE
sorted list of categories and editting mode alphabetically

### PR DESCRIPTION
**Pull request by DwinaTech**

### CODE REVIEW

File: frontend/src/Components/Organisation/OrganisationForm.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import helpers from '../../helpers';
 import categoriesData from '../../Data/Categories.json'
 
 const days = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday', 'Mon-Fri'];
-const areas = ['South', 'North', 'West', 'East', 'Central', 'All'];
+const areas = ['South', 'North', 'West', 'East', 'Central', 'All'].sort();
```
Comment (kkarimi) [inline frontend/src/Components/Organisation/OrganisationForm.js]: You don't need `.sort` here this is static data, just arrange it yourself

Comment (DwinaTech) [reply frontend/src/Components/Organisation/OrganisationForm.js]: Removed

File: frontend/src/Components/Organisation/OrganisationForm.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ import helpers from '../../helpers';
 import categoriesData from '../../Data/Categories.json'
 
 const days = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday', 'Mon-Fri'];
-const areas = ['South', 'North', 'West', 'East', 'Central', 'All'].sort();
+const areas = ['South', 'North', 'West', 'East', 'Central', 'All'];
```
Comment (kkarimi) [inline frontend/src/Components/Organisation/OrganisationForm.js]: You still need to manually sort these in alphabetical order! Just put them in order yourself

File: frontend/src/Components/Organisation/OrganisationForm.js (hunk 1 above)
Comment (kkarimi) [reply frontend/src/Components/Organisation/OrganisationForm.js]: @DwinaTech Just removing is not enough, did you read the comment above? Arrange it yourself

File: frontend/src/Components/Organisation/OrganisationForm.js (hunk 2 above)
Comment (DwinaTech) [reply frontend/src/Components/Organisation/OrganisationForm.js]: Done

**Pull request merged**